src/components/flow/index.js

```diff
@@ -179,7 +186,17 @@ const Flow = forwardRef(
       onClose?.();
     }, [flowWriter, onClose]);
 
-    const togglePlayback = useCallback(() => setIsPlayback((p) => !p), []);
+    const togglePlayback = useCallback(() => {
+      setIsPlayback((p) => {
+        setNerdletState((prevState) => ({
+          flow: {
+            ...prevState.flow,
+            isPlayback: !p,
+          },
+        }));
```

Comment:
This is going to trigger 2 state changes when `isPlayback` is set to true (once here, and another time in your `useEffect` where you check for changes to `isPlayback` in `nerdletState`). You could just set the `isPlayback` in `nerdletState` here, and let the `useEffect` trigger the state change.